packages/read-model/tests/Unit/Image/ImageViewFactoryTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ReadModelBundle\Unit\Image;
+
+use PHPUnit\Framework\TestCase;
+use Shopsys\FrameworkBundle\Component\Image\Image;
+use Shopsys\ReadModelBundle\Image\ImageView;
+use Shopsys\ReadModelBundle\Image\ImageViewFactory;
+
+class ImageViewFactoryTest extends TestCase
+{
+    public function testCreateFromImage(): void
+    {
+        $imageMock = $this->createMock(Image::class);
```

Comment:
It's sad we have to use mocks in unit tests :-( ... but I understand the reason. Right now I don't have a good feeling about the `Image` entity but it's not your fault